samples/Snippets/Snippets.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Polly.Core" />
+    <PackageReference Include="Polly.RateLimiting" />
+    <PackageReference Include="Polly.Extensions" />
+    <PackageReference Include="Polly.Testing" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" />
+    <PackageReference Include="xunit" />
+  </ItemGroup>
```

Comment:
Maybe it would be better to reference these from source rather than NuGet? Then if new features are added in the future, you can add docs in the same PR, rather than having to wait until it's been published.

Reply:
I was thinking about this, but it would require adding sources to `Samples` project and based on ou current flow we are updating the code after the nuget is pushed anyway. 

Alternatively, move Snippets under `src` folder?

Reply:
Yeah I think moving the snippets somewhere else is better, as even though we might be doing lots of work now, post v8 it's probably going to be much less frequent in terms of pushes to NuGet.

Reply:
Are you fine putting them in `src/Snippets` ?

Reply:
Sure.